data/presets/highway/ladder.json

```
@@ -0,0 +1,32 @@
+{
+    "icon": "temaki-crossing_markings-ladder",
+    "fields": [
+        "name",
+        "incline_steps",
+        "access_simple",
+        "handrail",
+        "step_count",
+        "surface",
+        "width",
+        "height"
+    ],
+    "moreFields": [
+        "covered_no",
+        "level_semi",
+        "lit",
+        "oneway",
+        "ref"
+    ],
+    "geometry": [
+        "vertex",
+        "point",
+        "line"
```

```suggestion
        "vertex",
        "line"
```

I think we should only allow points on lines and lines. Not single points.

---

sure, you're right